docs/benchmarks.md

```diff
@@ -0,0 +1,57 @@
+# GCSFuse Performance Benchmarks
+
+FIO is used to perform load tests on GCSFuse. Below tables shows performance metrics of GCSFuse for different workloads for the given test setup:
+
+## Test setup:
+
+* Infra: GCP VM
+* VM Type: n2-standard-48
+* VM Bandwidth: 32Gbps
+* OS: debian-11
+* VM location: us-west1-c
+* GCS Bucket location: us-west1
+* Framework: FIO
+
+### FIO Spec
+* Test runtime: 60sec
+* Thread count: 40
+* Block Size: 1MB
+
+### GCSFuse command
+```
+gcsfuse  --max-conns-per-host=100 --implicit-dirs --stat-cache-ttl=60s 
+--type-cache-ttl=60s --disable-http2 <bucket-name> ~/gcs
+```
+## Reads
+### Sequential reads
```

Comment:
Can we give the command as well similar to https://cloud.google.com/compute/docs/disks/benchmarking-pd-performance#existing-disk ?

Reply:
done